### PR TITLE
[ticket/16147] Updated tokens legend in BBCodes ACP

### PR DESCRIPTION
**Pull request by JoshyPHP**
PHPBB3-16147

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16147

### CODE REVIEW

File: phpBB/language/en/acp/posting.php [hunk 1]
```diff
@@ -78,13 +78,13 @@
 	'TOO_MANY_BBCODES'		=> 'You cannot create any more BBCodes. Please remove one or more BBCodes then try again.',
 
 	'tokens'	=>	array(
-		'TEXT'			=> 'Any text, including foreign characters, numbers, etc… You should not use this token in HTML tags. Instead try to use IDENTIFIER, INTTEXT or SIMPLETEXT.',
```
Comment (3D-I) [inline phpBB/language/en/acp/posting.php]: Does that means is possible to use {TEXT} token inside a HTML attribute now?

Comment (JoshyPHP) [reply phpBB/language/en/acp/posting.php]: It's always been possible and as far as I went back in the repository's history, it's always been safe to use. I think the warning dates back to a very old version of phpBB.

Comment (3D-I) [reply phpBB/language/en/acp/posting.php]: Thx, that was my impression as well.

Comment (3D-I) [reply phpBB/language/en/acp/posting.php]: There are though check-in and language strings still like `BBCODE_DANGER` and the logic in `includes/acp/acp_bbcodes.php`

```
				else
				{
					confirm_box(false, $user->lang['BBCODE_DANGER'], build_hidden_fields(array_merge($hidden_fields, array(
						'action'				=> $action,
						'bbcode'				=> $bbcode_id,
						'bbcode_match'			=> $bbcode_match,
						'bbcode_tpl'			=> htmlspecialchars($bbcode_tpl),
						'bbcode_helpline'		=> $bbcode_helpline,
						'display_on_posting'	=> $display_on_posting,
						)))
					, 'confirm_bbcode.html');
				}
```

**Comment by marc1706** [comment]
Closed and reopened to have travis run tests on the 3.2.x base.

**Pull request merged**